Doc/library/sqlite3.rst

```diff
@@ -98,6 +71,32 @@ This example uses the iterator form::
    ('2006-04-05', 'BUY', 'MSFT', 1000, 72.0)
 
 
+.. _sqlite3-placeholders:
+
+Usually your SQL operations will need to use values from Python variables.  You
+shouldn't assemble your query using Python's string operations because doing so
+is insecure; it makes your program vulnerable to an SQL injection attack
+(see https://xkcd.com/327/ for a humorous example of what can go wrong)::
+
+   # Never do this -- insecure!
+   symbol = 'RHAT'
+   cur.execute("SELECT * FROM stocks WHERE symbol = '%s'" % symbol)
+
+Instead, use the DB-API's parameter substitution. Put a placeholder wherever
+you want to use a value, and then provide a tuple of values as the second
+argument to the cursor's :meth:`~Cursor.execute` method. A statement may use
```

Comment:
minor nit: since this is the first mention of "statement", maybe specify "SQL statement" to avoid confusion
```suggestion
argument to the cursor's :meth:`~Cursor.execute` method. A SQL statement may use
```

Reply:
Good point, but in that case, I'd like to change the article as well. IIRC, the pronunciation of _SQL_ is _ess cue el_, which calls for an _an_. Thanks :)

Reply:
Fixed in 1482f21209a9b8ddd34705a59964dd046e406990

Reply:
Wow, I read it as "sequel". TIL from Google though that pronouncing it "sequel" is unofficial, and "ess-cue-ell" is the official version. So you're right about the "an"!

Reply:
Yeah, I've been saying "sequel" for years! 😅 I learned only last year that the "correct" pronunciation is to spell the letters out one by one.

Reply:
And FYI: The official pronunciation of SQLite is "ess-cue-ell-ite"! :)